devel/README

```diff
@@ -0,0 +1,2 @@
+This directory contains helpers for the project developers.
+Everyone else can safely ignore it.
```

Comment:
This file should be added to EXTRA_DIST in devel/Makefile.am

Reply:
Thanks, updated

Reply:
Should we distribute gdbhelpers as well?

Reply:
Added gdbhelpers to EXTRA_DIST because why not, and tweaked devel/Makefile.am a bit